events/message.go

```diff
@@ -111,6 +111,10 @@ type Bounce struct {
 	TransmissionID  string            `json:"transmission_id"`
 }
 
+func (b *Bounce) EventType() string {
```

Comment:
good point. EventCommon already implements the interface so this is redundant, will update.